src/main/java/org/apache/commons/codec/digest/DigestUtils.java

```diff
@@ -192,6 +192,7 @@ public static MessageDigest getDigest(final String algorithm) {
      */
     public static MessageDigest getDigest(final String algorithm, final MessageDigest defaultMessageDigest) {
         try {
+            /** potential insecure algorithm (MD2, MD5, SHA1, SHA256) called here */
```

Comment:
@YYTVicky 
This would better done in the method Javadoc for the algorithm param.

Reply:
Hi @garydgregory , Thanks a lot for your kind reply, we are a security research team at Virginia Tech. We are doing an empirical study about the usefulness of the existing security vulnerability detection tools. May we follow up on several questions: 
 **Is the bug report helpful?**
**will you prefer to fix the reported vulnerability**
**Are there any types of bugs/security vulnerabilities you want the detection tools to pay more attention to?**
Appreciate any feedback from your side!

Reply:
"Is the bug report helpful?"
No because it ignores documented behavior, I consider it a false positive.